core/ledger/pvtdatastorage/store.go

```diff
@@ -58,8 +58,6 @@ type Store interface {
 	Prepare(blockNum uint64, pvtData []*ledger.TxPvtData, missingPvtData ledger.TxMissingPvtDataMap) error
 	// Commit commits the pvt data passed in the previous invoke to the `Prepare` function
 	Commit() error
-	// Rollback rolls back the pvt data passed in the previous invoke to the `Prepare` function
-	Rollback() error
```

Comment:
Line 28 in this file references this function in it's comment, needs to be removed

Reply:
Thanks @btl5037. Good catch. I removed all the references to term "rollback" in the comments.